### ISSUE TITLE
Blank Region and Language Settings

### ISSUE THREAD

**Issue by RPGRoger** [pop-os/cosmic-settings#590]
Distribution:
NAME="Pop!_OS"
VERSION="24.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 24.04 LTS"
VERSION_ID="24.04"
HOME_URL="https://pop.system76.com/"
SUPPORT_URL="https://support.system76.com/"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=noble
UBUNTU_CODENAME=noble
LOGO=distributor-logo-pop-os

Issue/Bug Description:
Installed Pop!_OS 24.04 alpha on an Lenovo ideapad 320 via USB flash drive and Ventoy with Swedish as language and Swedish default keyboard layout. When I open the Region and Language settings it's blank.

Steps to reproduce:
Do a clean installation of Pop!_OS, choose Swedish as language and Swedish/Svenska default keyboard layout. After installation and you see your desktop, click on the COSMIC Settings icon via the dock panel, go to "Tid & språk" and click on "Region & språk".

Other Notes:
I updated and upgraded the system through the terminal and restarted the computer - the issue remains.

![screenshot-2024-09-14-09-40-06](https://github.com/user-attachments/assets/cf22f4f9-b78d-4f38-b3dc-89a1f6ce6153)

**Comment by l-const** [comment]
Yeah, it is not implemented yet @RPGRoger , this is a duplicate of https://github.com/pop-os/cosmic-settings/issues/12 .

**Comment by RPGRoger** [comment]
Aha, thanks!
